diff --git a/config.def.h b/config.def.h
index b41747f..78a6bdf 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "unifont:pixelsize=16";
 static int borderpx = 2;
 
 /*
@@ -438,7 +438,5 @@ static uint selmasks[] = {
  * of single wide characters.
  */
 static char ascii_printable[] =
-	" !\"#$%&'()*+,-./0123456789:;<=>?"
-	"@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_"
-	"`abcdefghijklmnopqrstuvwxyz{|}~";
+	"a";
 
diff --git a/st.c b/st.c
index 2594c65..6f4ec03 100644
--- a/st.c
+++ b/st.c
@@ -3301,7 +3301,7 @@ xloadfont(Font *f, FcPattern *pattern)
 	f->rbearing = f->match->max_advance_width;
 
 	f->height = f->ascent + f->descent;
-	f->width = DIVCEIL(extents.xOff, strlen(ascii_printable));
+	f->width = strlen(ascii_printable) == 1 ? extents.width : DIVCEIL(extents.xOff, strlen(ascii_printable));
 
 	return 0;
 }
